tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/GitHubLabelsTool/GitHubLabelsTool.cs

```diff
@@ -0,0 +1,228 @@
+        private readonly Option<string> documentationLinkOpt = new(["--link", "-l"], "Brand documentation link used to create a PR for a new label.") { IsRequired = true };
+
+        private readonly Argument<string> _serviceLabelArg = new Argument<string>(
+            name: "service-label",
+            description: "The service label to check in the common labels CSV"
+        )
+        {
+            Arity = ArgumentArity.ExactlyOne // only one service label is expected
+        };
+
+        public override Command GetCommand()
+        {
+            var command = new Command("github-labels", "GitHub service labels tools");
+            var subCommands = new[]
+            {
+                new Command(checkServiceLabelCommandName, "Check if a service label exists in the common labels CSV") { _serviceLabelArg },
+                new Command(createServiceLabelCommandName, "Creates a PR for a new label given a proposed label and brand documentation.") { proposedServiceLabelOpt, documentationLinkOpt },
+            };
+
+            foreach (var subCommand in subCommands)
+            {
+                subCommand.SetHandler(async ctx => { await HandleCommand(ctx, ctx.GetCancellationToken()); });
+                command.AddCommand(subCommand);
+            }
+            return command;
+        }
+
+        public override async Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            var command = ctx.ParseResult.CommandResult.Command.Name;
+            var commandParser = ctx.ParseResult;
+
+            switch (command)
+            {
+                case checkServiceLabelCommandName:
+                    var serviceLabel = commandParser.GetValueForArgument(_serviceLabelArg);
+                    var result = await CheckServiceLabel(serviceLabel);
+                    ctx.ExitCode = ExitCode;
+                    output.Output(result);
+                    return;
+                case createServiceLabelCommandName:
+                    var proposedServiceLabel = commandParser.GetValueForOption(proposedServiceLabelOpt);
+                    var documentationLink = commandParser.GetValueForOption(documentationLinkOpt);
+                    var createdPRLink = await CreateServiceLabel(proposedServiceLabel, documentationLink ?? "");
+                    output.Output($"Create service label result: {createdPRLink}");
+                    return;
+                default:
+                    SetFailure();
+                    output.OutputError($"Unknown command: '{command}'");
+                    return;
+            }
+        }
+
+        [McpServerTool(Name = "CheckServiceLabel"), Description("Checks if a service label exists in the common-labels.csv and returns its details if found.")]
+        public async Task<string> CheckServiceLabel(string serviceLabel)
+        {
+            try
+            {
+                return (await getServiceLabelInfo(serviceLabel)).ToString();
```

Comment:
This is fine as long as we don't want to return a string that provides more context about the results.

Reply:
I was wrong about this. The tools should [return structured data](https://github.com/Azure/azure-sdk-tools/blob/main/tools/azsdk-cli/README.md?plain=1#L226). So do something like 

```csharp
var serviceLabelStatus = await getServiceLabelInfo(serviceLabel);
return new { Status = serviceLabelStatus; LabelName = serviceLabel };
```